types/vtex.js/vtex.js-tests.ts

```diff
@@ -0,0 +1,18 @@
+import vtexjs = require('vtex.js')
```

Comment:
AFAIK this won't ever work. Does it work for you if you use plain JS?
I could be wrong and reviewed wrong module, in that case please show us source code for module compatible with CJS/ESM/UMD

Reply:
I think you could be right, I only use this module in native stores, and this script came by default on the store, so I use directly 'vtexjs.checkout' and by window 'window.vtexjs' but I don't find a way to test like this.

Reply:
You should be able to just not do an import in the test. You may need to flip tsconfig back to `commonjs` instead of `node16`, though.